docs/source/manual/configuration.rst

```diff
@@ -673,9 +673,9 @@ type                         REQUIRED                                   The appe
 currentLogFilename           REQUIRED                                   The filename where current events are logged.
 threshold                    ALL                                        The lowest level of events to write to the file.
 queueSize                    256                                        The maximum capacity of the blocking queue.
-discardingThreshold          51                                         When the blocking queue has only the capacity mentioned in discardingThreshold
+discardingThreshold          -1                                         When the blocking queue has only the capacity mentioned in discardingThreshold
                                                                         remaining, it will drop events of level TRACE, DEBUG and INFO, keeping only events
-                                                                        of level WARN and ERROR. If no discarding threshold is specified, then a default
+                                                                        of level WARN and ERROR. If no discarding threshold is specified (-1), then a default
```

Comment:
> then a default of queueSize / 5 is used

It took me a while to track down the origin of this. It should be called out that this is based on logback's implementation and the divisor may be subject to change.

https://github.com/qos-ch/logback/blob/v_1.2.3/logback-core/src/main/java/ch/qos/logback/core/AsyncAppenderBase.java#L104-L105

Reply:
Sounds good, I callout that the ratio is defined by logback.